### PR TITLE
[backfills] wip

### PR DESCRIPTION
**Pull request by vguerci**
This is an ugly piece of code that kinda works to create backfills, not intended for merge, but to review & discuss.

### CODE REVIEW

File: core/src/main/javascript/app/pages/BackfillCreate.js [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+    let { classes, workflow, back } = this.props;
+    let { selectedJobs } = this.state;
+
+    return (
+      <Window title="Create Backfill">
+        <CloseIcon className={classes.close} onClick={back} />
+        <form onSubmit={this.createBackfill}>
+          <FancyTable key="properties">
+            <dt key="name_">Name:</dt>
+            <dd key="name"><input ref="_name" type="text" required /></dd>
```
Comment (guillaumebort) [inline core/src/main/javascript/app/pages/BackfillCreate.js]: I don't think you need to keep a reference to these form elements. The state of the main component should contain their data, and you can you onChange event handlers to handle state changes.

File: core/src/main/javascript/app/pages/BackfillCreate.js [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+            }
+            onChange={this.selectJobs}
+          />
+        </div>
+      </Window>
+    );
+  }
+}
+
+// TODO duplicated styles
```
Comment (guillaumebort) [inline core/src/main/javascript/app/pages/BackfillCreate.js]: Yes I'm not sure what is this style for :)

File: core/src/main/javascript/app/pages/BackfillCreate.js (hunk 1 above)
Comment (vguerci) [inline core/src/main/javascript/app/pages/BackfillCreate.js]: done, also improved a bit using HTML5 datetime inputs, will change this to a text input

File: core/src/main/javascript/app/pages/BackfillCreate.js (hunk 2 above)
Comment (vguerci) [inline core/src/main/javascript/app/pages/BackfillCreate.js]: Cleaned up styles, but still some duplication / hacks

**Comment by vguerci** [comment]
Abandoned, see #65 instead.

**Pull request closed without merging**